src/AWS.Logger.Core/Core/AWSLoggerCore.cs

```diff
@@ -79,16 +79,28 @@ public AWSLoggerCore(AWSLoggerConfig config, string logType)
             _config = config;
             _logType = logType;
 
-            var credentials = DetermineCredentials(config);
-
-            if (_config.Region != null)
-            {
-                _client = new AmazonCloudWatchLogsClient(credentials, Amazon.RegionEndpoint.GetBySystemName(_config.Region));
-            }
-            else
-            {
-                _client = new AmazonCloudWatchLogsClient(credentials);
-            }
+            if (!string.IsNullOrEmpty(_config.ServiceUrl))
+            {
+                var awsConfig = new AmazonCloudWatchLogsConfig() { ServiceURL = _config.ServiceUrl };
+                if (_config.ServiceUrl.StartsWith("http://", StringComparison.OrdinalIgnoreCase))

```

Comment:
For consistency we should put brackets for each of these clauses. I typically only skip brackets if the condition only contains a single return statement.

Reply:
Fixed